blockchain/blockchain.go

```diff
@@ -611,6 +611,12 @@ func (bc *blockchain) MintNewBlock(
 			ActionGasLimit: bc.config.Genesis.ActionGasLimit,
 			Registry:       bc.registry,
 		})
+
+	if newblockHeight == bc.config.Genesis.AleutianBlockHeight {
```

Comment:
not sure if I understand correctly, only the delegate who mint the aleutian block will perform the below update, others won't update new value 15K into admin state?

i think we need a special 'governance' type action to update 15K, and in commitBlock() everyone take that action to update to 15K? that seems a lot of change, but consider in the future we may run into similar change

or we should pass AleutianHeight to rewarding protocol, same as what we did for evm protocol?

Reply:
> others won't update new value 15K into admin state?

Validator will update too

> that seems a lot of change, but consider in the future we may run into similar change

That's true, and it's not going to be a lot adjustment. Hence, it's not worthwhile yet of making an rewarding governance protocol

Reply:
agreed, not worthwhile for now